be2-scala/src/main/scala/ch/epfl/pop/decentralized/Monitor.scala

```diff
@@ -66,7 +66,7 @@ final case class Monitor(
 }
 
 object Monitor {
-  def props(heartbeatGenRef: ActorRef, heartbeatRate: FiniteDuration = 30.seconds, messageDelay: FiniteDuration = 3.seconds): Props =
+  def props(heartbeatGenRef: ActorRef, heartbeatRate: FiniteDuration = 15.seconds, messageDelay: FiniteDuration = 1.seconds): Props =
```

Comment:
Is it okay if I deal with that in pr #1519 ? It might be worth it to add that in the config file so there is no need to recompile everytime we would want to change that setting ?